tests/foreman/api/test_contentview.py

```diff
@@ -1,4 +1,5 @@
 # -*- encoding: utf-8 -*-
+# pylint: disable=too-many-public-methods, too-many-lines, invalid-name
```

Comment:
I see this kind of disabling in many modules, why not creating a pylint config file? At least for global disables.